### PR TITLE
Aspell

### PR DESCRIPTION
**Pull request by oktalz**

### CODE REVIEW

File: aspell.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+	var data []byte
+	var err error
+	if data, err = os.ReadFile(filename); err != nil {
+		log.Printf("warning: aspell exceptions file not found (%s)", err)
+	}
+	words = strings.Split(string(data), "\n")
+
+	return words, nil
+}
```
Comment (oliwer) [inline aspell.go:66]: Rename function to `NewAspellWithExceptions()` or something with "New".

**Review by oliwer — COMMENTED** [review]
Just some wording changes.

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,11 @@
-FROM golang:alpine as builder
+FROM golang:alpine AS builder
 RUN mkdir /build
 ADD . /build/
 WORKDIR /build
 RUN go build -o check
 
 FROM alpine:latest
+RUN apk --no-cache add aspell
```
[the remark this answers is not in the document]
Comment (oktalz) [reply Dockerfile:8]: we do not, but I'll add them just to be 'future safe' if that changes, in both ubuntu and alpine i had it installed automatically

File: aspell.go (hunk 1 above)
Comment (oktalz) [reply aspell.go:66]: I'll add a package, it is a small app, but regardless its cleaner

**Pull request merged**